apps/vr-tests/src/stories/Button/Button.stories.tsx

```diff
@@ -8,8 +8,8 @@ const baseProps: IButtonProps = {
   iconProps: {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31997&commitId=a820b3a799685a4777f575816a2bfe3c6c8a196d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button (compat)</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button (compat).Checked.default.chromium.png|1760| Changed|
|Button (compat).Checked.pressed.chromium.png|1789| Changed|
|Button (compat).Checked.hover.chromium.png|1789| Changed|
|Button (compat).Primary Disabled.default.chromium.png|1619| Changed|
|Button (compat).Primary Disabled.hover.chromium.png|1619| Changed|
|Button (compat).Primary Checked.hover.chromium.png|1480| Changed|
|Button (compat).Disabled.default.chromium.png|1667| Changed|
|Button (compat).Primary Checked.default.chromium.png|1480| Changed|
|Button (compat).Primary.hover.chromium.png|1489| Changed|
|Button (compat).Disabled.hover.chromium.png|1667| Changed|
|Button (compat).Primary.pressed.chromium.png|1474| Changed|
|Button (compat).Root - RTL.default.chromium.png|1846| Changed|
|Button (compat).Disabled.pressed.chromium.png|1667| Changed|
|Button (compat).Root.hover.chromium.png|1775| Changed|
|Button (compat).Root.pressed.chromium.png|1760| Changed|
|Button (compat).Root.default.chromium.png|1785| Changed|
|Button (compat).Primary Disabled.pressed.chromium.png|1619| Changed|
|Button (compat).Root - RTL.hover.chromium.png|1835| Changed|
|Button (compat).Primary.default.chromium.png|1491| Changed|
|Button (compat).Root - RTL.pressed.chromium.png|1818| Changed|
|Button (compat).Primary Checked.pressed.chromium.png|1480| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>